modules/openapi-generator/src/main/resources/swift5/api.mustache

```diff
@@ -275,7 +275,7 @@ extension {{projectName}} {
         case http{{code}}(value: {{#dataType}}{{{dataType}}}{{/dataType}}{{^dataType}}Void{{/dataType}}, raw: ClientResponse)
```

Comment:
@aymanbagabas I think only the status code should have 200 should have something like this
If it's 400 we don't know the response type.
```
case http200(value: Double, raw: ClientResponse)
case http400(value: ????, raw: ClientResponse)
```

Reply:
Per the [specs](https://swagger.io/docs/specification/describing-responses/#default), you can absolutely define responses for any status code. Maybe I should remove the value and only return `ClientResponse` when the response is empty instead of returning `Void` for the value.

Reply:
That's how it is right now, when the response in the specs is empty, it returns `Void` for the value. As long as the specs contain a response schema, it would have that in the value. I will change it to `ClientResponse` only when the response is empty aka no `dataType` for the response.

Reply:
I think now it put's the success response in all status code's, or am I wrong?

Reply:
No, the Petstore specs are different, for examples `2_0/petstore.yaml` doesn't have a 200 response schema on POST `/pet` while the `3_0` version does. Thus, the generated code for `2_0` doesn't have a 200 response enum case for POST `/pet` aka `addPet` operation.